traffic_router/core/src/main/java/com/comcast/cdn/traffic_control/traffic_router/core/loc/RegionalGeo.java

```diff
@@ -357,7 +355,12 @@ public static void enforce(final TrafficRouter trafficRouter, final Request requ
         if (result.getType() == DENIED) {
             routeResult.setResponseCode(result.getHttpResponseCode());
         } else {
-            routeResult.addUrl(new URL(createRedirectURIString(httpRequest, deliveryService, cache, result)));
+                final String redirectURIString = createRedirectURIString(httpRequest, deliveryService, cache, result);
```

Comment:
nit: this block appears to be over-indented

Reply:
fixed